### PR TITLE
fix: fix getitem by ellipsis

### PR DESCRIPTION
**Pull request by JoanFM**
Goals:

- ...
- ...
- [ ] check and update documentation, if required. See [guide](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#documentation-guidelines)

### CODE REVIEW

**Comment by JohannesMessner** [comment]
Can we add this syntax to the table [here](https://docarray.jina.ai/fundamentals/documentarray/access-attributes/#attribute-selector)?

**Review by JohannesMessner — COMMENTED** [review]
Can we add this syntax to the table [here](https://docarray.jina.ai/fundamentals/documentarray/access-attributes/#attribute-selector)?

**Comment by JoanFM** [comment]
> Can we add this syntax to the table [here](https://docarray.jina.ai/fundamentals/documentarray/access-attributes/#attribute-selector)?

Done! please recheck!

File: docs/fundamentals/documentarray/access-attributes.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ As in element selector, one can use attribute selector to **get/set/delete** att
 
 | Example                                      | Return                                                                                                                        |
 |----------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|
-| `da[:, 'id']`                                | all `.id` in a List                                                                                                           |
+| `da[:, 'id']`                                | all `.id` from all root Documents in a List                                                                                   |
+| `da[..., 'id']`                              | all `.id` from all flatten Documents in a List                                                                                |
```
Comment (JohannesMessner) [inline docs/fundamentals/documentarray/access-attributes.md:22]:
```suggestion
| `da[..., 'id']`                              | all `.id` from all flattened Documents (root, chunks, and matches) in a List                                                                                |
```

**Review by JohannesMessner — APPROVED** [review]

**Pull request merged**